src/config/dynamic-theme-fixes.config

```diff
@@ -14099,14 +14099,14 @@ CSS
 notion.so
 
 INVERT
-img[alt="People using Notion"]
-.global-margin-s .logos
-.persona-grid > .persona-grid-item > .persona-grid-image
-.desktop-illustration
-.illustration .next-image
-.api-section-image .next-image
-.visual .next-image
+.next-image
 .logo svg > :first-child
+.notranslate::after
+.logo-limit-height
+.jsx-2604928324.mobile-actions
+.jsx-fb3129685d85c2fa.menu-toggle
+.jsx-422720195.logo
+.jsx-1042831150.logo
```

Comment:
The `.jsx-xxxxx` ones should be removed, they are randomized and generally doesn't last long before they are replaced by the site.